lib/utils/createConfig.js

```diff
@@ -117,10 +122,12 @@ function createConfig(config, argv, { port }) {
     options.lazy = true;
   }
 
+  // TODO remove in `v4`
```

Comment:
Why will we remove this from v4?

Reply:
@hiroppy in favor `logLevel` option or `logger`, we should refactor this place because we have `4` option for output log and stats, i think we should have 1/2 option, so i mark this as TODO on removing, anyway we can revisit this in future

Reply:
Thank you for answering. I understood.